wp-config.php

```diff
@@ -8,48 +8,46 @@
  * https://pantheon.io/docs
  */
 
+/**
+ * Pantheon platform settings. Everything you need should already be set.
+ */
+if (file_exists(dirname(__FILE__) . '/wp-config-pantheon.php') && isset($_ENV['PANTHEON_ENVIRONMENT'])) {
+	require_once(dirname(__FILE__) . '/wp-config-pantheon.php');
+
 /**
  * Local configuration information.
  *
  * If you are working in a local/desktop development environment and want to
  * keep your config separate, we recommend using a 'wp-config-local.php' file,
  * which you should also make sure you .gitignore.
  */
-if (file_exists(dirname(__FILE__) . '/wp-config-local.php') && !isset($_ENV['PANTHEON_ENVIRONMENT'])):
-  # IMPORTANT: ensure your local config does not include wp-settings.php
-  require_once(dirname(__FILE__) . '/wp-config-local.php');
+} elseif (file_exists(dirname(__FILE__) . '/wp-config-local.php') && !isset($_ENV['PANTHEON_ENVIRONMENT'])){
```

Comment:
You don't need both an `elseif` and the `!isset`.

I don't like else, so I'd prefer to have redundant isset / !isset.  Alternately, just skip the !isset here. On the Drupal side, the local settings file is included if present even if PANTHEON_ENVIRONMENT is set.  It's a little safer to check PANTHEON_ENVIRONMENT, though, so I'm fine with keeping that check in.